releasenotes/notes/1.1/prepare-1.1.0-732e167b70ab4d01.yaml

```
@@ -0,0 +1,52 @@
+    The highlights for this release are:
+
+      * Support for typed classical variables has been added to Qiskit's :class:`.QuantumCircuit`. These
+        classical variables can be specified as inputs or as scoped variables in a :class:`.QuantumCircuit`
+        where they e.g. store the output of qubit measurements or target control-flow operations. Support
+        for e.g. setting gate parameters or output variables will be added in the future.
+
+      * The default two qubit synthesis methods that are used internally by the transpiler in the
+        :class:`.UnitarySynthesis` pass have been re-implemented in Rust. This yields significant
+        runtime speedups when decomposing two qubit unitary matrices. As a consequence, the runtime of
+        transpilation with optimization level 3 was significantly improved where running
+        :class:`.UnitarySynthesis` incurred a large runtime overhead historically. This release also
+        starts running :class:`.UnitarySynthesis` as part of the optimization stage in optimization level
+        2 because of these runtime performance improvements.
+
+        Additionally, the numeric methods used in :class:`.Isometry` have been moved to rust,
+        enabling large runtime speed-ups in particular for controlled unitary gate synthesis.
+        The decomposition for multi-controlled :class:.XGate` and :class:`.PhaseGate`  has been
+        improved resulting in a reduction in the number of gates used in the synthesis by more
+        than two orders of magnitude.
+
+      * A number of new transpiler passes have been introduced to Qiskit that yield significant runtime
+        speedups while also decreasing the size of the transpiled quantum circuits in many cases. Specifically,
+        :class:`.ElidePermutations` and :class:`.StarPreRouting` have been demonstrated to have a significant
+        impact on the routing output quality and runtime and :class:`.RemoveFinalReset` can improve quantum circuits
+        that include resets.
+
+      * The default pass managers have been improved by extending them with the newly introduced transpiler
+        passes. In particular, the optimization level 2 preset pass manager from
+        :class:`.generate_preset_pass_manager` and used internally by :func:`.transpile` has been refactored to
+        have a better tradeoff between runtime and optimization effort in order to serve as a default pass
+        manager in future releases. While this release doesn't change the default to use level 2 it is a better
+        choice than using level 1 or 3.
+
+      * New generic primitive V2 implementations were added, :class:`~.BackendEstimatorV2` and
+        :class:`~.BackendSamplerV2`, to compliment the existing full statevector based implementations.
+
+
+      * Changes to platform support: python 3.8 is deprecated starting with Qiskit 1.1.0 and will no longer be supported in 1.3.0,
```

Previous release notes capitalize Python

```suggestion
      * Changes to platform support: Python 3.8 is deprecated starting with Qiskit 1.1.0 and will no longer be supported in 1.3.0,
```